src/Microsoft.DotNet.Wpf/src/PresentationFramework/Resources/xlf/Strings.zh-Hans.xlf

```diff
@@ -4785,7 +4785,7 @@ Do you want to replace it?</source>
       </trans-unit>
       <trans-unit id="ScrollBar_ContextMenu_PageDown">
         <source>Page Down</source>
-        <target state="translated">向下翻页</target>
+        <target state="translated">Page Down</target>
```

Comment:
I do not think it is a good change. The original code is Chinese, but the new code is English